### ISSUE TITLE
[Feature Request]: Eloquent Model Attribute Autocompletion on `$model->touch()` method

### ISSUE THREAD

**Issue by ralphjsmit** [laravel-idea/plugin#778]
### Feature Description

I updated some of my Eloquent model calls that only updated a timestamp to the `touch()` version:

Before:

```php
$license->update([
   'some_attribute_at' => now(),
]);
```

After:

```php
$license->touch('some_attribute_at');
```

I noticed there was no autocompletion on the `touch()` method. Minor thing, but would be nice to have :) Thanks!

**Comment by adelf** [comment]
Implemented in 7.3.

**Comment by ralphjsmit** [comment]
Thanks!
